src/main/common/utils/numericUtils.ts

```diff
@@ -1,5 +1,21 @@
+import * as numeral from 'numeral'
 import * as _ from 'lodash'
 
+function convert (strVal: string) {
+  // checks length, followed by a filter by not containing ',' or valid thousand separators
+
+  if (strVal.length > 0) {
+
+    if ((!strVal.includes(',') || containsAThousandSeparator(strVal))) {
+      return numeral(strVal).value()
+    } else {
+      return _.toNumber(strVal)
```

Comment:
I would suggest using just one tool to convert string to number. Having two methods might be read as something we missed that when corrected will have unexpected side-effect.

If numeral library is giving us surprising result we might return undefined early when unsupported case is reached.